samples/csharp_dotnetcore/02.echo-bot/Startup.cs

```diff
@@ -35,22 +35,24 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
             }
-            else
-            {
-                app.UseHsts();
-            }
 
-            app.UseDefaultFiles();
-            app.UseStaticFiles();
-            app.UseWebSockets();
-            //app.UseHttpsRedirection();
-            app.UseMvc();
+            app.UseDefaultFiles()
```

Comment:
The OOTB AspNet.Core template adds an app.UseEndpoints() call in startup.cs is that not needed?

Sample:
```
            app.UseEndpoints(endpoints =>
            {
                endpoints.MapControllers();
            });
```